LibreHardwareMonitor/Utilities/HttpServer.cs

```diff
@@ -672,46 +676,46 @@ private string GeneratePrometheusResponse(Node node)
                     tagName = tagName.ToLower();
 
                     // Preparing the labels for all data and uniqueness
-                    string valueId = sensor.Sensor.Identifier.ToString();
+                    string valueSensorId = sensor.Sensor.Identifier.ToString().Substring(valueHardwareId.Length);
 
-                    string _nameKey = $"{valueHardware}.{valueSensor}.{tagSensorType}{tagSensorUnits}";
+                    string _nameKey = $"{valueHardwareAlias}_{valueSensorName}_{tagSensorType}{tagSensorUnits}";
                     //Have we seen this path before?
-                    if (_prometheusPaths.ContainsKey(valueId))
+                    if (_prometheusIdName.ContainsKey(valueSensorId))
                     {
                         //Yes, then use the name.
-                        valueSensor = _prometheusPaths[valueId];
+                        valueSensorAlias = _prometheusIdName[valueSensorId];
                     }
                     else
                     {
                         //No, but have we seen the sensor name under this same parent and tagName?
-                        if (_prometheusNames.ContainsKey(_nameKey))
+                        if (_prometheusNameCount.ContainsKey(_nameKey))
                         {
                             //Yes, we have seen it. Append the counter and increment it.
-                            valueSensor += $" {_prometheusNames[_nameKey].ToString()}";
-                            _prometheusNames[_nameKey] += 1;
+                            valueSensorAlias += $" {_prometheusNameCount[_nameKey]}";
+                            _prometheusNameCount[_nameKey] += 1;
                         }
                         else
                         {
                             //No, we have not seen it, so store it.
-                            _prometheusNames.Add(_nameKey, 1);
+                            _prometheusNameCount.Add(_nameKey, 1);
                         }
                         //Save the name to the path
-                        _prometheusPaths.Add(valueId, $"{valueSensor}");
+                        _prometheusIdName.Add(valueSensorId, $"{valueSensorName}");
                     }
 
                     string valueFamily = node.Nodes[i].Text;
                     string valueHost = _root.Text;
 
                     // Creates the tag with labels
-                    string tagLine = $$"""{{tagName}} {"sensor"="{{valueSensor}}" "hardware"="{{valueHardware}}" "id"="{{valueId}}" "parentId"="{{hardwarePath}}" "family"="{{valueFamily}}" "host"="{{valueHost}}"}""";
+                    string tagLine = $$"""{{tagName}} {"sensorName"="{{valueSensorName}}" "sensorAlias"="{{valueSensorAlias}}" "hardwareName"="{{valueHardwareName}}" "hardwareAlias"="{{valueHardwareAlias}}" "sensorId"="{{valueSensorId}}" "hardwareId"="{{valueHardwareId}}" "host"="{{valueHost}}"}""";
```

Comment:
I do not like the idea of having `hardwareAlias` be `{valueHardwareName} ({valueHardwareId})` and `sensorAlias` be `{valueSensorName} ({valueSensorId})`

Pros: It requires no dictionaries, all the naming logic can be removed.
Cons: it modifies all the aliases not just the duplicates, can't be modified downstream, some ids are really long (e.g. 
/throughput/7/nic/%7B2DCDD9D7-D331-466D-926F-D6C957B2D917%7D)

But if that is the only way for this PR to go through then I will do that because I would rather have the data in Prometheus rather than not having it.

Reply:
Ok, made it so it is `name (id)`

This should be ready to merge.